hybridCarboniteMetadataService/src/test/scala/cromwell/services/metadata/hybridcarbonite/CarboniteWorkerActorSpec.scala

```diff
@@ -20,6 +21,13 @@ class CarboniteWorkerActorSpec extends TestKitSuite("CarboniteWorkerActorSpec")
   val ioActor = TestProbe()
   val carboniteFreezerActor = TestProbe()
 
+  val fasterBackOff: SimpleExponentialBackoff = SimpleExponentialBackoff(
+    initialInterval = 1.second,
```

Comment:
I'd even go as far as making this `initialInterval = 10.millis, maxInterval = 100.millis` since you don't ever expect this to actually be waiting very long